docs/en/contributing/install.md

```diff
@@ -6,6 +6,10 @@ sidebar_label: Install
 
 # Install
 
+## Prerequisites
+
+Before setting up FAST-DNA, install Git and [npm](https://www.npmjs.com/get-npm).
```

Comment:
Should we link to git installation site as well?

Reply:
Since this lives on github, I don't think so; we're being fairly generous in calling out that git is a prerequisite here IMO. There are several different ways to install and use git, including using the github app. If someone does not know how to use git or how to install it, we should not point them to a specific way or flavor; it is easy enough to search how to do so and find a solution that works.

I think it makes sense to call out and link to NPM because there is now a competitor in `yarn` and it is important to ensure that you have the correct package manager installed. Additionally, `npm` has a single source of truth which handles all scenarios for ensuring you have it installed and describing what it is.

Reply:
cc: @awentzel as I see you asked the same question as well

Reply:
I don't think it costs anything to add a link to [here](https://git-scm.com/doc) (I do not agree we should provide link to any specific type of installation or config).  It makes it faster for those that need the link because it's their first time on git and doesn't hurt our content at all since it's only a link and doesn't change the content otherwise.  I am sending MIS/CS students to our repository and 50% of them have not used git and it's useful to them among others.

Reply:
I think we need to delineate between contributing to FAST and consuming FAST. Git not required to install our packages, or to use them in a project. NPM is required. 

As an aside, non-collaborators *should not* directly clone our package - they will not be able to push. They should fork and clone their own instance. If this documentation is because we think people are trying to "use" our stuff, our documentation is confusing. I would not recommend us pushing people to the repo in order to "play" with the packages. That's likely better done using something like "create-react-app" and adding the packages as a dependency.

If this is specific to *contributing*, git is required and while I want to encourage anyone to contribute (I also shared this with the students at Code Fellows on Friday), there is a certain expectation that you can clone/fork the repo yourself (via terminal, or the github app, etc) and you have a way to manage that. If you can't there are significant resources online (including in the repo itself via a Github button) to help someone get up and running.

My primary problem with the link you provided is that there is a very clear "downloads" link that exists which drives people to specific flavors and ways to use git.

Either way, I'd be fine pointing to documentation if there were a place that either didn't include specific flavors, or documented/aggregated all the various ways you can use git.

Reply:
Possibly, although I think that works in the context of a FULL tutorial. It's specific to what they are trying to accomplish there that they are including that. Here we are directly within the context of contributing to the source repository, and there is an expectation that you are responsible for your development environment.

Reply:
Leaving out the link for now given that this is the contributors section. If we come up with a decent generic solution we can re-visit but I would agree that the reminder should be good enough for anyone who fits the profile of this section. I asked similar questions last night and I think the big issue is that we don't have a regular getting started section yet so folks are thinking about this like this is getting started, and its not.